features/contact/requesting_contact.feature

```diff
@@ -0,0 +1,28 @@
+@requesting_contact
+Feature: Requesting contact
+    In order to get help from the store's support team
```

Comment:
`to receive` and `support` is quite sufficient on its own, no need for the `team`